src/main/java/com/stuypulse/robot/subsystems/Climber.java

```diff
@@ -5,24 +5,155 @@
 
 package com.stuypulse.robot.subsystems;
 
+import java.io.Console;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.stuypulse.robot.Constants;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 /*-
  * Climbs at end of match
  *
  * Contains:
- *      - TBD
+ *      - Change tilt of climber
+ *      - Move climber via motor
+ *      - Different tilt angles
+ *      - Encoder + Solenoid used for stopping
  *
- * @author [TBD]
+ * @author independence106(Jason Zhou)
+ * @author Ca7Ac1(Ayan Chowdhury)
+ * @author marcjiang7(Marc Jiang)
+ * @author ambers7(Amber Shen)
+ * @author Souloutz(Howard Kong)
+ * @author jiayuyan0501(Jiayu Yan)
+ * @author ijiang05(Ian Jiang)
+ * @author TraceyLin (Tracey Lin)
+ * @author annazheng14(Anna Zheng)
+ * @author lonelydot(Raymond Zhang)
+ * @author andylin2004 (Andy Lin)
  */
 public class Climber extends SubsystemBase {
 
+    private Solenoid solenoidLong;
+    private Solenoid solenoidShort;
+
+    private DigitalInput limitSwitch;
+
+    private Solenoid stopper;
+
+    private RelativeEncoder encoder;
+
+    private CANSparkMax climber;
+
     public Climber() {
-        // Initialize Motors, etc... here!
+        climber = new CANSparkMax(Constants.Ports.Climber.MOTOR);
+        climber.setInverted(Constants.ClimberSettings.MOTOR_INVERTED);
+  
+        encoder = climber.getEncoder();
+        encoder.setPositionConversionFactor(Constants.ClimberSettings.CIRCUMFERENCE * Constants.ClimberSettings.GEAR_RATIO);
```

Comment:
You can store the gear ratio with the circumference multiplied. If people see CIRCUMFERENCE * (1/36.0) then they'll know what the ratio is. 

or even CONVERSION_FACTOR = CIRCUMFERENCE * GEAR_RATIO and u use conversion_factor 

This also isn't make or break, but a lot of the other gear ratios do this